src/util/resource_manager.cpp

```diff
@@ -222,8 +192,8 @@ uint64_t ResourceManager::getTimeUsage() const { return d_cumulativeTimeUsed; }
 
 uint64_t ResourceManager::getResourceRemaining() const
 {
-  if (d_resourceBudgetCumulative <= d_cumulativeResourceUsed) return 0;
-  return d_resourceBudgetCumulative - d_cumulativeResourceUsed;
+  if (options::cumulativeResourceLimit() <= d_cumulativeResourceUsed) return 0;
```

Comment:
Instead of calling options:: here, can we already do this properly by caching the reference to options passed to this manager and looking it up on that object?

Reply:
Same goes for the rest of the occurrences of `options::` in this file.

Reply:
As per offline discussion: before doing this, we want to do some major refactoring of the options class. I'd just leave it as it is for now.